pkg/antctl/raw/traceflow/command.go

```diff
@@ -36,22 +37,16 @@ import (
 
 	"antrea.io/antrea/pkg/antctl/raw"
 	"antrea.io/antrea/pkg/apis/crd/v1alpha1"
+	antrea "antrea.io/antrea/pkg/client/clientset/versioned"
 )
 
 const defaultTimeout time.Duration = time.Second * 10
 
 var (
-	Command *cobra.Command
-	option  = &struct {
-		source      string
-		destination string
-		outputType  string
-		flow        string
-		liveTraffic bool
-		droppedOnly bool
-		timeout     time.Duration
-		nowait      bool
-	}{}
+	Command      *cobra.Command
+	getClients   = getk8sClient
+	newTraceflow = getNewTraceflow
```

Comment:
I believe you don't need this method variable as well. You can pass right options to get your desired traceflow or error. In this case, you don't even need a `TestGetNewTraceflow` to test `getNewTraceflow`. It should be covered when you test `runE` if you provide correct parameters.

Reply:
getNewTraceflow needs to be faked because phase and status are not present in traceflow object created by actual method, which cause failure while getting the traceflow and causes timeout.
By faking this method i created a traceflow with phase and status already present in traceflow object.